README.md

```diff
@@ -1 +1,23 @@
-# portfolio
\ No newline at end of file
+# Project Name:portfolio
+This is my first project at Microverse.
+## Built With
+.HTML
+.CSS
+## Getting Started
+to implement and run this project you should have the following prerequisites  
+## Prerequisites
+Text editor
+browser like crome
+## Author
+ [Githab](https://github.com/settings/profile)
+[linkedin](https://www.linkedin.com/in/haftamu-desta-795791a1/)
+[tiwtter](https://twitter.com/DestaHftamu?t=NQ4ovkdWbsfsjh62NFEXFg&s=09)
+
+## Contributing
+Contributions, issues, and feature requests are welcome!
+## Show your support
+give tumps if you like this project.
```

Comment:
- [ ] Please avoid typos in your README because it is supposed to explain to someone how your project works and any form of typos can mislead and confuse someone on how to work with your project 👍 . Instead of 'tumps', you could write 'thumbs up' 😎